### ISSUE TITLE
Suggestion for better API compatability!

### ISSUE THREAD

**Issue by Harry989** [drizzle-mizzle/Character-Engine-Discord#31]
Hey there! I love the look of this project and this could easily become my favourite discord bot. The only issue is API compatibility, for example I would like to use Ollama as my API service.

Integrating a library called LiteLLM (https://github.com/BerriAI/litellm) would easily allow you to interface with many llm api's and improve compatibility tenfold. I hope you could take this into consideration, many thanks for reading!

**Comment by jakobdylanc** [comment]
LiteLLM is python so I doubt they'd use it here

**Comment by drizzle-mizzle** [comment]
Yeah, python stuff has no use here, but I have a good news for you. Right now I'm rebuilding the bot from scratch with APIs' modularity prioritized, and I have two weeks vacation starting next week, finally.

**Comment by Harry989** [comment]
> Yeah, python stuff has no use here, but I have a good news for you. Right now I'm rebuilding the bot from scratch with APIs' modularity prioritized, and I have two weeks vacation starting next week, finally.

That’s really great to hear! I’ll make sure to star and follow this project! I’m always excited to come across cool stuff.
